app/models/book.rb

```diff
@@ -195,6 +195,20 @@ def self.from_marcxml_record(record:, key:)
     book
   end
       
+  def as_message
+    {"SQS message" => "New or Updated Record for Book: #{self.title}"}
```

Comment:
I see, so essentially we want the Gateway to consume the message `(as_message`) with a `book's properties` in json format. Since we have a method that puts it in json format already (`as_json()`), I can call on that inside the `as_message` method like so? 

```ruby
def as_json(options = { })
    # existing code here
end 

def as_message 
   self.as_json 
end 
```